setup.cfg

```diff
@@ -33,7 +33,7 @@ install_requires=
     importlib_metadata  ; python_version < "3.10"
     joblib
     matplotlib ~=3.0
-    numba ~=0.56.0
+    numba >=0.56
```

Comment:
You wrote that we need at least numba 0.57, is that not correct anymore?

Reply:
For 3.11, 3.10 and below is fine with 0.56

Reply:
And since versions 0.56 correctly specifies that it is not compatible with 3.11, there is no issue.

The conda-forge package for 0.57 is not yet out, so prescribing 0.57 is not a good idea.